apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/CharSequenceToContainCreatorSamples.kt

```diff
@@ -0,0 +1,219 @@
+        expect("ABBBCD") toContain o atLeast 1 the values("A", "B", "C", "D")
+
+        fails {
+            expect("AAAAAABBBB") toContain o atMost 3 the values("A", "B")
+        }
+        fails {
+            expect("AAABBBB") toContain o exactly 3 the values("A", "B")
+        }
+        fails {
+            expect("AAAAAABBBB") toContain o atLeast 3 the values("A", "B", "C")
+        }
+    }
+
+    @Test
+    fun valuesIgnoringCase() {
+        expect("AbC") toContain o ignoring case the values("A", "B", "c")
+
+        fails {
+            expect("aabaabbb") toContain o ignoring case the values("A", "B", "C")
+        }
+    }
+
+    @Test
+    fun valuesIgnoringCaseWithChecker() {
+        // `values` is a final step in the CharSequence.toContain expectation-building process and can be used with
+        // various checkers (see CharSequenceToContainCheckerSamples)
+
+        expect("ABc") toContain o ignoring case exactly 1 the values("A", "b", "C")
+        expect("AaaBC") toContain o ignoring case atMost 3 the values("A", "B", "c")
+        expect("ABBBcD") toContain o ignoring case atLeast 1 the values("a", "b", "C", "d")
+
+        fails {
+            expect("AAAAAABBBB") toContain o ignoring case atMost 3 the values("a", "b")
+        }
+        fails {
+            expect("AAABBBB") toContain o ignoring case exactly 3 the values("A", "b")
+        }
+        fails {
+            expect("AAAAAABBBB") toContain o ignoring case atLeast 3 the values("a", "b", "C")
+        }
+    }
+
+
+    @Test
+    fun regex() {
+        // `regex` is a final step in the CharSequence.toContain expectation-building process and can be used with
+        // various checkers (see CharSequenceToContainCheckerSamples)
+
+        expect("ABC") toContain o exactly 1 regex "A(BC|C/DC)"
+        expect("AAABC") toContain o atMost 3 regex "A(BC|C/DC)"
+        expect("ABBBCD") toContain o atLeast 1 regex "A(BC|C/DC)"
+
+        fails {
+            expect("AAAAAABBBB") toContain o atMost 3 regex "A(BC|C/DC)"
+        }
+        fails {
+            expect("AAABBBB") toContain o exactly 3 regex "A(BC|C/DC)"
+        }
+        fails {
+            expect("AAAAAABBBB") toContain o atLeast 3 regex "A(BC|C/DC)"
+        }
+    }
+
+    @Test
+    fun regexIgnoringCase() {
+        expect("AbC") toContain o ignoring case regex "A(BC|C/DC)"
+
+        fails {
+            expect("aabaabbb") toContain o ignoring case regex "A(BC|C/DC)"
+        }
+    }
+
+    @Test
+    fun regexIgnoringCaseWithChecker() {
+        // `regex` is a final step in the CharSequence.toContain expectation-building process and can be used with
+        // various checkers (see CharSequenceToContainCheckerSamples)
+
+        expect("ABc") toContain o ignoring case exactly 1 regex "A(BC|C/DC)"
+        expect("AaaBC") toContain o ignoring case atMost 3 regex "A(BC|C/DC)"
+        expect("ABBBcD") toContain o ignoring case atLeast 1 regex "A(BC|C/DC)"
```

Comment:
this one fails, see https://github.com/robstoll/atrium/actions/runs/9523378291/job/26270699966?pr=1774#step:5:799
There are also other failures